Dockerfile: Erase unnecesarry packages

giovanism opened the pull request:
Erased packages :
```
aaa_base
dbus-1
dbus-1-x11
gio-branding-openSUSE
kmod
libthai-data
lksctp-tools
logrotate
openssh
openslp
perl-Net-DBus
postfix
R-core-doc
sysconfig-netconfig
systemd-presets-branding-openSUSE
wicked-service
```
Related to github.com/coala/docker-coala-base/issues/55

jayvdb commented:
Also try removing the other two dbus packages.  If they are needed, just add a comment here with a link to the broken build.

Commit:
Dockerfile: Erase unnecessary packages

The removed packages are the following:

aaa_base
dbus-1
dbus-1-x11
gio-branding-openSUSE
kmod
libthai-data
lksctp-tools
logrotate
openssh
openslp
perl-Net-DBus
postfix
R-core-doc
sysconfig-netconfig
systemd-presets-branding-openSUSE
wicked-service

Related to https://github.com/coala/docker-coala-base/issues/55

jayvdb commented:
ack 5e427cb

jayvdb commented:
@rultor merge

rultor commented:
> @rultor merge

@jayvdb OK, I'll try to merge now. You can check the progress of the merge [here](http://www.rultor.com/t/10620-272641682)

rultor commented:
> @rultor merge

@jayvdb Done! FYI, the full log is [here](http://www.rultor.com/t/10620-272641682) (took me 1min)
